app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt

```diff
@@ -50,7 +50,7 @@ class NewsRVAdapter : RecyclerView.Adapter<NewsRVAdapter.NewsViewHolder>() {
         holder.binding.apply {
             titleTv.text = newsList[position].title
             authorTv.text = newsList[position].byline
-            if(newsList[position].media[0].mediaMetadata[0].url.isNullOrEmpty())
+            if(!newsList[position].media.isNullOrEmpty() || !newsList[position].media[0].mediaMetadata.isNullOrEmpty())
```

Comment:
@martin371998 already commented in other PR

Reply:
done